src/deploy-to-network.ts

```diff
@@ -102,3 +128,31 @@ export async function phase0(): Promise<void> {
 
   await provider.close();
 }
+
+export async function vestedTokens(): Promise<void> {
+  console.log("The wallet owner will be the one providing tokens for vesting");
+  const provider = await walletConnectProvider();
+  const signer = provider.getSigner();
+  const token = await askForAddress("of the token");
+  const owner = await askForAddress("of the owner of all created vestings");
+  do {
+    const beneficiary = await askForAddress("of beneficiary");
+    const amount = await askForBigNumber("amount to vest");
+    const vestingStartTime = await askForBigNumber(
+      "vesting start time in seconds since epoch"
+    );
+    const vestingPeriod = await askForBigNumber("vesting period in seconds");
+    const cliffPeriod = await askForBigNumber("cliff period in seconds");
```

Comment:
Since everyone has the same cliff and vesting period, maybe we can have these as constants in the deploy script?

Reply:
Fixed